wallet-server-ui/src/app/component/new-user-onboarding/new-user-onboarding.component.html

```diff
@@ -0,0 +1,173 @@
+<div class="new-user-onboarding" @fadeOut>
+  <mat-sidenav-container>
+    <mat-sidenav-content>
+      <div class="sidenav__menu-container">
+        <div class="app-logo"></div>
+
+        <mat-action-list class="sidenav__menu-list">
+          <button
+            mat-list-item
+            *ngFor="let onboardingStep of onboardingStepCards"
+            (click)="onStepSelected(onboardingStep.stepNumber)"
+          >
+            <span
+              translate
+              class="material-icons-outlined sidenav__menu-list-item__icon"
+            >
+              {{ onboardingStep.menuIconKey }}
+            </span>
+
+            <span
+              translate
+              class="sidenav__menu-list-item__title"
+            >
+              {{ onboardingStep.menuTitleKey }}
+            </span>
+          </button>
+        </mat-action-list>
+      </div>
+    </mat-sidenav-content>
+  </mat-sidenav-container>
+
+  <div class="main-content-container">
+    <div [@stepSlider]="currentStep" class="card-slide-container">
+      <ng-container *ngFor="let onboardingStep of onboardingStepCards; let cardIndex = index">
+        <new-user-onboarding-card
+          [onboardingStep]="onboardingStep"
+          *ngIf="cardIndex == currentStep - 1"
+          class="card-slide-container__card"
+        >
+
+          <div class="card-slide-container__card__projected-content">
+
+            <!-- Step 1 -->
+
+            <button
+              class="card-slide-container__card__action-button"
+              *ngIf="onboardingStep.stepNumber == 1"
+              mat-flat-button
+              color="primary"
+              (click)="onImportExport()"
+            >
+              Back Up Now
```

Comment:
Move to translation key

Reply:
Resolved by 24e0bf481e989e782838ebfeccbb1400a717ddae